phpBB/bin/phpbbcli.php

```diff
@@ -73,6 +73,7 @@
 
 /** @var \phpbb\language\language $language */
 $language = $phpbb_container->get('language');
+$language->set_default_language($config['default_lang']);
```

Comment:
do not rely on the global: `$language->set_default_language($phpbb_container->get('config')['default_lang']);`